modules/ROOT/pages/users.adoc

```diff
@@ -22,32 +23,33 @@ This feature is not available in Anypoint Platform Private Cloud Edition.
 When running Anypoint Platform Private Cloud Edition, you need to configure user management methods such as SSO through a third-party IdP or LDAP.
 --
 
-As an organization administrator of a master organization or of one of its business groups, you can invite new users and manage existing users for your organization on the _Access Management_ Administration page, which you can reach by selecting the *Users* option in your *Access Management* section.
+As an organization administrator of a root organization or of one of its business groups, you can invite new users and manage existing users for your organization.
 
 To invite new users to your organization:
 
-. Click *Users* in the left navigation bar.
+include::partial$include-nav-users.adoc[]
+
 . Click *Invite user*.
 . Enter the email addresses of who you want to invite in your organization as a comma-separated list.
-. Optionally, select one or more xref:roles.adoc[roles] to which to assign these users.
+. Optionally, select a xref:teams.adoc[team] or xref:roles.adoc[role (deprecated)] to which to assign these users.
 . Click *Send Invitation*.
 
 The users who you invite receive email invitations to sign up to your organization. Invited users must use the link they receive in the invitation email to join your organization. When they click through the link they are presented with a sign-up form that already has the *Company* field completed, matching your organization.
 
 Note that the email link expires in one week.
 
-Invited users have access to the same set of resources as you (although they may have different roles that can restrict what they can view or do).
+Invited users have access to the same set of resources as you (although they may have different permissions that can restrict what they can view or do).
 
 == Resending or Canceling a Sent Invitation
 
 Click *Pending Invites* to view all invitations that have not yet been accepted. Select the ones you need to manage, and click *Re-send Invite* or *Cancel Invite*.
 
-[[granting-permissions-and-roles-to-users]]
-== Granting Permissions and Roles to Users
+[[grant-user-permissions]]
+== Grant Permissions and Roles to Users
 
-Click a user name to access more information about that user, add roles and permissions to it, or reset its password.
+Organization administrators can manage user access using xref:teams.adoc[teams], xref:roles.adoc[roles] (deprecated), or by granting permissions to users individually.
```

Comment:
To remove modal verb...:
```suggestion
Organization administrators manage user access using xref:teams.adoc[teams] or xref:roles.adoc[roles] (deprecated), or by granting permissions to users individually.
```
Same questions: having permissions vs. having that title/job; explanation of "deprecated"